src/test/shell/bazel/BUILD

```diff
@@ -1055,44 +1055,44 @@ sh_test(
     tags = ["no_windows"],
 )
 
-sh_test(
-    name = "bazel_bootstrap_distfile_test",
-    timeout = "eternal",
-    srcs = ["bazel_bootstrap_distfile_test.sh"],
-    args = [
-        "$(location //:bazel-distfile)",
-        "$(location //src:embedded_jdk_allmodules)",
-    ],
-    data = [
-        ":test-deps",
-        "//:bazel-distfile",
-        "//src:embedded_jdk_allmodules",
-        "@bazel_tools//tools/bash/runfiles",
-    ],
-    exec_compatible_with = ["//:highcpu_machine"],
-    tags = ["block-network"],
-)
-
-sh_test(
-    name = "bazel_bootstrap_distfile_tar_test",
-    timeout = "eternal",
-    srcs = ["bazel_bootstrap_distfile_test.sh"],
-    args = [
-        "$(location //:bazel-distfile-tar)",
-        "$(location //src:embedded_jdk_allmodules)",
-    ],
-    data = [
-        ":test-deps",
-        "//:bazel-distfile-tar",
-        "//src:embedded_jdk_allmodules",
-        "@bazel_tools//tools/bash/runfiles",
-    ],
-    exec_compatible_with = ["//:highcpu_machine"],
-    tags = [
-        "block-network",
-        "no_windows",
-    ],
-)
+# sh_test(
```

Comment:
And here.

Reply:
I'll just leave these as broken since they shouldn't ever land this way